game/world/objects/npc.cpp

```diff
@@ -1832,72 +1833,55 @@ void Npc::tickTimedEvt(Animation::EvCount& ev) {
 
   for(auto& i:ev.timed) {
     switch(i.def) {
-      case ZenLoad::DEF_NULL:
-      case ZenLoad::DEF_LAST:
-        break;
-      case ZenLoad::DEF_CREATE_ITEM: {
+      case phoenix::mds::event_tag_type::create_item: {
         if(auto it = invent.addItem(i.item,1,world())) {
           invent.putToSlot(*this,it->clsId(),i.slot[0]);
           }
         break;
         }
-      case ZenLoad::DEF_INSERT_ITEM: {
+      case phoenix::mds::event_tag_type::insert_item: {
         invent.putCurrentToSlot(*this,i.slot[0]);
         break;
         }
-      case ZenLoad::DEF_REMOVE_ITEM:
-      case ZenLoad::DEF_DESTROY_ITEM: {
-        invent.clearSlot(*this,"",i.def!=ZenLoad::DEF_REMOVE_ITEM);
+      case phoenix::mds::event_tag_type::remove_item:
+      case phoenix::mds::event_tag_type::destroy_item: {
+        invent.clearSlot(*this, "", i.def != phoenix::mds::event_tag_type::remove_item);
         break;
         }
-      case ZenLoad::DEF_PLACE_ITEM:
+      case phoenix::mds::event_tag_type::place_item:
         break;
-      case ZenLoad::DEF_EXCHANGE_ITEM: {
+      case phoenix::mds::event_tag_type::exchange_item: {
         if(!invent.clearSlot(*this,i.slot[0],true))
           invent.clearSlot(*this,"",true); // fallback for cooking animations
         if(auto it = invent.addItem(i.item,1,world())) {
           invent.putToSlot(*this,it->clsId(),i.slot[0]);
           }
         break;
         }
-
-      case ZenLoad::DEF_FIGHTMODE:
+      case phoenix::mds::event_tag_type::fight_mode:
         break;
-      case ZenLoad::DEF_PLACE_MUNITION: {
+      case phoenix::mds::event_tag_type::place_munition: {
         auto active=invent.activeWeapon();
         if(active!=nullptr) {
-          const int32_t munition = active->handle().munition;
+          const int32_t munition = active->handle()->munition;
           invent.putAmmunition(*this,uint32_t(munition),i.slot[0]);
           }
         break;
         }
-      case ZenLoad::DEF_REMOVE_MUNITION: {
+      case phoenix::mds::event_tag_type::remove_munition: {
         invent.putAmmunition(*this,0,"");
         break;
         }
-      case ZenLoad::DEF_DRAWSOUND:
```

Comment:
please don't remove empty cases. Idea here to have explicit view on what is not yet implemented